### PR TITLE
Code Review component

### PR DESCRIPTION
**Pull request by maureensturgeon**
Creates a code review component to display the review session and comments. The icons are different than the mocks because the icons specified in the mocks are not available to us.

**Open code review:**
Screen Shot 2022-04-29 at 4 58 51 PM

**Closed code review:**
Screen Shot 2022-05-02 at 11 44 10 AM

## Links

- jira ticket: [LP-2301](https://codedotorg.atlassian.net/browse/LP-2301)

## Testing story
Tested locally and added unit tests

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import {codeReviewTimelineElementType} from '@cdo/apps/templates/instructions/codeReviewV2/CodeReviewTimelineElement';
+import javalabMsg from '@cdo/javalab/locale';
+import Comment from '@cdo/apps/templates/instructions/codeReview/Comment';
+
+const DEFAULT_REVIEW = {
+  id: 1,
+  createdAt: '2022-03-31T04:58:42.000Z',
+  isClosed: false,
+  projectVersion: 'asdfjkl',
+  isVersionExpired: false
```
Comment (hannahbergam) [inline apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx:14]: This looks pretty great to me! Do you have a test case for isVersionExpired: true? Or is that unnecessary?

File: apps/src/templates/instructions/codeReviewV2/CodeReviewTimelineReview.jsx [hunk 2]
```diff
@@ -3,22 +3,128 @@ import PropTypes from 'prop-types';
...
+            comment={comment}
+            key={`code-review-comment-${comment.id}`}
+            onResolveStateToggle={() => {}}
+            onDelete={() => {}}
+            viewAsCodeReviewer={true}
+          />
+        ))}
+        {!isClosed && (
+          <div style={{border: '1px solid black'}}>
+            Comment editor placeholder
```
Comment (hannahbergam) [inline apps/src/templates/instructions/codeReviewV2/CodeReviewTimelineReview.jsx:61]: Is this meant to be like a 'type comment here' text? I can't quite tell by the name.

**Review by hannahbergam — COMMENTED** [review]
Looks good! Just a couple questions:)

File: apps/src/templates/instructions/codeReviewV2/CodeReviewTimelineReview.jsx (hunk 2 above)
Comment (maureensturgeon) [reply apps/src/templates/instructions/codeReviewV2/CodeReviewTimelineReview.jsx:61]: This is for a component that I haven't built yet, it'll be the comment editor with the code insert functionality
Screen Shot 2022-05-02 at 3 25 53 PM

File: apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx (hunk 1 above)
Comment (maureensturgeon) [reply apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx:14]: I have the unit test `passes project version and is version expired to CodeReviewTimelineElement` which will check that the right prop is passed through

File: apps/src/templates/instructions/codeReviewV2/CodeReviewTimelineReview.jsx (hunk 2 above)
Comment (hannahbergam) [reply apps/src/templates/instructions/codeReviewV2/CodeReviewTimelineReview.jsx:61]: Ohh got it. Thanks for explaining!

File: apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx (hunk 1 above)
Comment (hannahbergam) [reply apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx:14]: What happens when isVersionExpired is true?

**Review by jamescodeorg — APPROVED** [review]
Looks great!

File: apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx (hunk 1 above)
Comment (maureensturgeon) [reply apps/test/unit/templates/instructions/codeReviewV2/CodeReviewTimelineReviewTest.jsx:14]: I'll add another test!

**Pull request merged**